utkdigitalinitiatives/iiif_assemble
# Address Digital-1419: Modify descriptive metadata for RFTA Artists.

# What Does This Do

1. Adds a `not(@encoding)` rule to date mappings.
2. Maps specific related resources to `Related Resource`.

# How Can I Test

Take your favorite piece of art and do something like this. 

https://digital.lib.utk.edu/assemble/manifest/rftaart/1?update=1

You can use see it in a viewer if it's helpful here following same pattern:

https://uv-v4.netlify.app/#?manifest=https://digital.lib.utk.edu/assemble/manifest/rftaart/1

Keep in mind that the way this is styled in a viewer does not necessarily mean it will be that way for us (nectar).
